Custom Function: New - Indicator Remove Tag

kelby-shelton opened the pull request:
Added a new custom function for removing indicator tags off of an indicator record

kelby-shelton committed:
Custom Function: New - Indicator Remove Tag

Added a new custom function for removing indicator tags off of an indicator record

philroyer-phantom commented on custom_functions/indicator_remove_tag.json:
```diff
@@ -0,0 +1,40 @@
+        {
+            "contains_type": [],
+            "data_path": "indicator_tags",
+            "description": "The new tags for the indicator"
```
I think this should be something like "The remaining tags on the indicator after the modification"

philroyer-phantom replied:
I think this should be something like "The remaining tags on the indicator after the modification"

philroyer-phantom commented on custom_functions/indicator_remove_tag.py:
```diff
@@ -0,0 +1,65 @@
+    data = {"tags": new_tags}
+    response = phantom.requests.post(url, json=data, verify=False).json()
+    if response.get('success'):
+        outputs = {'indicator_id': indicator_id, 'indicator_tags': tags}
```
I think this should be `new_tags`, not `tags`.

philroyer-phantom replied:
I think this should be `new_tags`, not `tags`.

philroyer-phantom reviewed (changes requested):
Two minor changes.

kelby-shelton committed:
Minor changes

- Adjusted output description
- Fixed output path to use new_tags instead of input tags.

philroyer-phantom reviewed (approved):
👍
